src/pydocstyle/cli.py

```diff
@@ -49,7 +49,7 @@ def run_pydocstyle():
         return ReturnCode.invalid_options
 
     count = 0
-    for error in errors:
+    for error in errors:  # type: ignore
```

Comment:
Why?

Reply:
Hiding the `error` variable defined above in `except IllegalConfiguration as error:`. I could rename it if you want.

Reply:
```
py37-tests runtests: commands[1] | mypy --config-file=tox.ini src/
src\pydocstyle\cli.py:52: error: Assignment to variable 'error' outside except: block
```